flow/CompressionUtils.cpp

```diff
@@ -128,4 +128,24 @@ TEST_CASE("/CompressionUtils/gzipCompression") {
 
 	return Void();
 }
+
+TEST_CASE("/CompressionUtils/gzipCompression2") {
```

Comment:
Is the intention of the test to validate "fully compressible string"? If yes, can we add some asserts about expected compressed string size?

Reply:
Yes, I asserted that is was smaller than the uncompressed string. We could be even more strict if we wanted, but the actual compressed size depends on the compression algorithm and level, so for example if we changed the default level of compression the test might also have to change.

Reply:
I think given we are populating string with single character, it would be safe to assume that resultant string would be <50% of original string (also given, size of string is 512-1KB, so overhead accounted) OR may be more... Just a thought..

Reply:
Sure I can change that